app/components/dropdown-search.js

```diff
@@ -24,6 +24,16 @@ export default Ember.Component.extend({
     return this.get("addAble") ? this.get("addItem") : "";
   }),
 
+  didUpdate() {
+    this._super(...arguments);
+    if (this.isDuplicateField) {
```

Comment:
we should _never_ refer to specific features in generic/reusable components !
Plus, this piece of code doesn't seem to really be doing anything

Reply:
So this is basically done to reset the dropdown values to null when the page is reloaded

Reply:
And in order to protect the reusability i have passed params IsDuplicate for it

Reply:
```
let dropDownValues = {
        ...this.get("dropDownValues")
      };
```

This wouldn't set anything to null 🤔 

Bottom line is, we can't have any reference to the duplicate feature in this file. You can probably trigger that from the controller somehow